playbooks/templates/.github/workflows/qemu-kvm-integration-tests.yml

```diff
@@ -133,7 +133,7 @@ jobs:
         run: |
           set -euo pipefail
           for f in tests/*.log; do
-              if FAIL=$(grep -B100 -A30 "fatal:" "$f"); then
+              if FAIL=$(grep -E -B100 -A30 "fatal:|An exception occurred" "$f"); then
```

Comment:
https://github.com/linux-system-roles/auto-maintenance/blob/main/check_logs.py#L253 `get_errors_from_ansible_log` does this and more.  We just need to add an option to check_logs.py to print the ansible errors in a 'github action' friendly way.